src/styles/_catalog.scss

```
@@ -1,22 +1,37 @@
+$cardWidth: 200px;
+
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  grid-template-columns: repeat(4, $cardWidth);
+  gap: 48px;
+
+  justify-content: center;
+
+  @media (max-width: 1023.5px) {
```

Try to use integers, not fractions
```suggestion
  @media (max-width: 1023) {
```